src/mds/Server.cc

```diff
@@ -4214,8 +4177,49 @@ void Server::handle_client_getattr(const MDRequestRef& mdr, bool is_lookup)
     }
   }
 
-  if (!mds->locker->acquire_locks(mdr, lov))
+  if (!mds->locker->acquire_locks(mdr, lov)) {
```

Comment:
Could you explain this change @lxbsz? Seems like this is more than just removing misleading logs?

Reply:
Oh, sorry for late. This just a mistake. I will remove it.